include/kernel/mem/boot.h

```diff
@@ -0,0 +1,62 @@
+#ifndef __KERNEL_MEM_BOOT_H__
+#define __KERNEL_MEM_BOOT_H__
```

Comment:
```suggestion
#ifndef __KERNEL__MEM_BOOT_H__
#define __KERNEL__MEM_BOOT_H__
```

`__NAMESPACE__FILE_H__`